canonical-web-and-design/jaas.ai
# Upgrade site to Vanilla Framework v2.1.0

## Done

Upgrade site to Vanilla Framework v2.1.0

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- Run through the site and check that everything looks as expected

## Details

Fixes #370
